src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -64,16 +57,17 @@
 import java.util.Collection;
 import java.util.List;
 import java.util.Map;
-import java.util.concurrent.CompletionStage;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.TimeUnit;
+import java.util.UUID;
+import java.util.concurrent.*;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import static com.emc.mongoose.item.op.OpType.NOOP;
+import static com.emc.mongoose.item.op.Operation.Status.*;
+import static com.emc.mongoose.storage.driver.pravega.PravegaConstants.*;
+import static com.emc.mongoose.storage.driver.pravega.io.StreamScaleUtil.scaleToFixedSegmentCount;
+
 public class PravegaStorageDriver<I extends Item, O extends Operation<I>>
-extends CoopStorageDriverBase<I, O>  {
+		extends CoopStorageDriverBase<I, O> {
```

Comment:
Please rollback all autoformatting related changes. These changes pollute the diff